hnn_core/parallel_backends.py

```diff
@@ -84,7 +84,8 @@ def _gather_trial_data(sim_data, net, n_trials, postproc):
 def _get_mpi_env():
     """Set some MPI environment variables."""
     my_env = os.environ.copy()
-    if 'win' not in sys.platform:
+    # For Linux systems
+    if sys.platform not in ['win32', 'darwin']:
```

Comment:
I'm not sure if the original intent of this line was to just target Linux systems, but that was what it was doing using the `'win' not in` logic. This has the same functionality but is more explicit of the platforms it avoids. 

The old logic would have also excluded the 'cygwin' platform, a posix-translator layer for windows... but I doubt anyone will be trying to use hnn-core on that.

Reply:
You'll have to look at Blake's old commits to understand why this flag was added. Likely a Windows-specific thing not related to Mac.

Reply:
I'm guessing this is intended only for unix systems which support open-mpi (vs. ms-mpi), in which case we should only be avoiding windows here.

Reply:
Also, setting the `OMPI_MCA_btl_base_warn_component_unused` environment variable to `0` is only used for suppressing warning messages. Regardless of Blake's original intention here, I suggest that we either implement this consistently for all references to an open-mpi build (i.e., across Mac and Linux systems) or not at all.

Reply:
To be clear, I'm suggesting we remove `'darwin'` from the list above or remove L87-89 altogether. Aside from that and @jasmainak's comments, this looks good to go by me.